crates/cxx-qt-lib/Cargo.toml

```diff
@@ -34,7 +34,8 @@ serde = { version = "1", features=["derive"], optional = true }
 cxx-qt-build.workspace = true
 
 [features]
-default = ["qt_gui", "qt_qml", "qt_quickcontrols"]
```

Comment:
can you add an entry in the `CHANGELOG.md` in the Changed section for this?

Reply:
sorted